projects/burnr/src/components/AccountCard.tsx

```diff
@@ -11,45 +11,42 @@ interface Props {
 }
 
 const Alert = (props: AlertProps) => {
-	return <MuiAlert elevation={6} variant="filled" {...props} />;
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
 }
 
 const AccountCard: React.FunctionComponent<Props> = ({ account, addressFormat }: Props) => {
-	const [showCopied, setShowCopied] = useState<boolean>(false);
-	return (
-		<Grid container wrap='nowrap' spacing={1} alignItems='center'>
-			<Grid item>
-				<Identicon
-					size={32}
-					theme='polkadot'
-					value={account.address}
-					onCopy={() => {
-						setShowCopied(true);
-						copyToClipboard(account.address);
-					}}
-				/>
-				<Snackbar
-					anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-					open={showCopied}
-					autoHideDuration={2000}
-					onClose={() => setShowCopied(false)}>
-					<Alert severity="success">Copied!</Alert>
-				</Snackbar>
-			</Grid>
-			<Grid item>
-				{ account.name !== '' &&
-			<Typography variant='h4'>
-				{account.name}
-			</Typography>
-				}
-				<Typography variant='subtitle2'>
-					{ addressFormat === 'Full'
-						? account.address
-						: account.address.slice(0,4) + '...' + account.address.slice(account.address.length - 4, account.address.length)
-					}
-				</Typography>
-			</Grid>
-		</Grid>
-	);};
+  const [showCopied, setShowCopied] = useState<boolean>(false);
+  return (
+    <>
+      <Snackbar
+        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+        open={showCopied}
+        autoHideDuration={2000}
+        onClose={() => setShowCopied(false)}>
+        <Alert severity="success">Copied!</Alert>
+      </Snackbar>
+      
+      <Box display='flex' alignItems='center'>
+        <Identicon
+          size={32}
+          theme='polkadot'
+          value={account.address}
+          onCopy={() => {
+            setShowCopied(true);
+            copyToClipboard(account.address);
+          }}
+        />
+        <Box height={32} display='flex' flexDirection='column' justifyContent='center' ml={1}>
+          { account.name !== '' && <Typography variant='h4'>{account.name}</Typography>}
```

Comment:
This is not correct. in React if the `account.name !== ''` is false then you will see a "false" printed in the UI.
Its better to:
```suggestion
          { account.name !== '' ? <Typography variant='h4'>{account.name}</Typography> : null}
```